app/controllers/api/orders_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::OrdersController < ApplicationController
+  
+  rescue_from ActiveRecord::RecordNotFound, with: :product_not_found
+
+  def create
+    product = Product.find(params[:order][:product_id])
+    order = Order.create
+    order.items.create(product: product, amount: params[:order][:amount])
+    render json: { message: "#{product.name} has been added to your order" }, status: 201
+  end
+
+  private
+   def product_not_found
+    render json: { message: "we could not find the product you tried to add." }
```

Comment:
I missed this when we were going through it: `render json:` defaults to an http status of 200 if no additional value is passed in. I suggest 404, this will need a corresponding test.